src/runtime/driver/mod.rs

```diff
@@ -343,7 +355,15 @@ impl Drop for Driver {
                     while self
                         .uring
                         .submission()
-                        .push(&AsyncCancel::new(id as u64).build().user_data(u64::MAX))
+                        .push(
+                            // Conversion to configured squeue::EntryMarker is useless when
+                            // sqe128 feature is disabled.
+                            #[allow(clippy::useless_conversion)]
+                            &AsyncCancel::new(id as u64)
+                                .build()
+                                .user_data(u64::MAX)
+                                .into(),
```

Comment:
The comment above helps, but I still don't know what this into() is converting to. I could clone the branch tomorrow and figure it out I guess.